test/unit/BypassUnsubscribeManagementTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * This file tests BypassUnsubscribeManagement.
+ */
+namespace SendGrid\Tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+use SendGrid\Mail\BypassUnsubscribeManagement;
+
+/**
+ * This file tests BypassUnsubscribeManagement.
+ *
+ * @package SendGrid\Tests
+ */
+class BypassUnsubscribeManagementTest extends TestCase
```

Comment:
It might be worthwhile to also add tests to validate that when both `bypass_list_management` and any of the three subfilters are set to true, the former overrides the latter